modules/manage/pages/security/authorization/acl.adoc

```diff
@@ -55,7 +55,19 @@ When specifying hosts, you must pair the `--allow-host` flag with the `--allow-p
 
 === Resources
 
-A resource is what an ACL allows or denies access to. There are four resources within Redpanda: `topics`, `groups`, `cluster`, and `transactionalid`. Names for each of these resources can be specified with their respective flags.
+A resource is what an ACL allows or denies access to. The following resources are available within Redpanda:
+
+- `topics`
+- `groups`
+- `cluster`
+- `transactionalid`
+
+Starting in version 25.2, Redpanda also supports the following ACL resources for Schema Registry:
+
+* **`REGISTRY`**: Controls whether or not to grant ACL access to global, or top-level Schema Registry operations. Specify using the flag `registry-global`.
+* **`SUBJECT`**: Controls ACL access for specific Schema Registry subjects. Specify using the flag `registry-subject`.
```

Comment:
are these supposed to be in caps? It seems weird that `SUBJECT` is uppercase and `groups` is lowercase.

Reply:
Or I guess we should just capitalize the other 4 here to be consistent